app/src/main/java/edu/uc/forbesne/shoppingsidekick/ui/main/MapsFragment.kt

```diff
@@ -83,38 +86,36 @@ class MapsFragment(store: Market) : Fragment() {
 
     private fun updateMap() {
         if (mapReady && locationDetails != null) {
-                if (!locationDetails.longitude.isEmpty() && !locationDetails.latitude.isEmpty()) {
-                    var currLatitude = locationDetails.latitude.toDouble()
-                    var currLongitude = locationDetails.longitude.toDouble()
-                    val marker = LatLng(currLatitude, currLongitude)
-                    var bottomBoundary = latitude.toDouble() - .005
-                    var leftBoundary = longitude.toDouble() - .005
-                    var topBoundary = latitude.toDouble() + .005
-                    var rightBoundary = longitude.toDouble() + .005
-                    if (currLongitude > longitude.toDouble()) {
-                        rightBoundary = currLongitude + .005
-                    }
-                    else {
-                        leftBoundary = currLongitude - .005
-                    }
-                    if (currLatitude > latitude.toDouble()) {
-                        topBoundary = currLatitude + .005
-                    }
-                    else {
-                        bottomBoundary = currLatitude - .005
-                    }
-
-
-                    var mapBoundary = LatLngBounds(LatLng(bottomBoundary, leftBoundary), LatLng(topBoundary, rightBoundary))
-                    mMap.addMarker(MarkerOptions().position(marker).title("Your location"))
-                    mMap.moveCamera(CameraUpdateFactory.newLatLngBounds(mapBoundary, 0))
-
-                    val milesDistance = getDistanceInMiles(latitude.toDouble(), longitude.toDouble(), currLatitude, currLongitude)
-                    txtDistance.text = "Distance: " + milesDistance.toString() + " miles"
-
+            if (!locationDetails.longitude.isEmpty() && !locationDetails.latitude.isEmpty()) {
```

Comment:
I just removed one extra indentation from this section of the code (I did not change anything).